tcs-service/src/main/resources/db/migration/data/hee/V4.24__delete_duplicatedPersonRecord.sql

```diff
@@ -0,0 +1,11 @@
+delete from RightToWork where id in(305827);
```

Comment:
Just wondering is the person id is `305827` or 265154? I think in the app you shared the page with the person id 265154.

Reply:
Yep so the one which will remain is `265154` which has more 'actual' data recorded against it